src/pages/HomePage/AllNotes/AllNotes.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Notes from 'src/components/UI/Notes/Notes';
+import { VAR_NotPinnedNotes, VAR_PinnedNotes } from 'src/utils/Route';
+import "../HomePage.css";
```

Comment:
Variable naming can be better and something like **PINNED_NOTES**, **NOT_PINNED_NOTES**